semgrep-core/tests/php/backed_enum.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
usually when the PR is only related to parsing, we just add the file in semgrep-core/tests/php/parsing/
Actually you'll see there was a semgrep-core/tests/php/parsing/TODO/enum.php in there.
No need to write a xxx.sgrep for it (unless the .sgrep actually tries to match an element in the construct 
that is now correctly parsed, for example here one case of the enum).